app/uk/gov/hmrc/gform/views/hardcoded/pages/save_with_access_code.scala.html

```diff
@@ -24,7 +24,7 @@
 )(implicit request: Request[_], messages: Messages, l: LangADT)
 
 @uk.gov.hmrc.gform.views.html.main_template(
-  title = saveWithAccessCode.heading,
+  title = "",
```

Comment:
`saveWithAccessCode.heading` seems to be harmless, since it originate in messages.